scripts/scil_visualize_gradients.py

```diff
@@ -21,16 +23,23 @@
                                           plot_each_shell,
                                           plot_proj_shell)
 
+sphere_choices = ['symmetric362', 'symmetric642', 'symmetric724',
+                  'repulsion724','repulsion100', 'repulsion200']
+
 
 def _build_arg_parser():
     p = argparse.ArgumentParser(
         formatter_class=argparse.RawDescriptionHelpFormatter,
         description=__doc__)
 
-    p.add_argument(
-        'in_gradient_scheme', nargs='+',
+    g = p.add_mutually_exclusive_group(required=True)
+    g.add_argument(
+        '--in_gradient_scheme', nargs='+',
         help='Gradient sampling filename. (only accepts .bvec and .bval '
              'together or only .b).')
+    g.add_argument(
+        '--dipy_sphere', choices=sphere_choices,
+        help="Sphere to show, amongst:\n{}".format(sphere_choices))
```

Comment:
don't need to add amongst because choices are already printed in the help. Apart from that good to go !